doc/sphinx/how-to/how-to-test-mir-for-a-release.md

```diff
@@ -0,0 +1,181 @@
+# How to test Mir for a release
+The following document details the criteria that must be met before we can
+publish an official release of Mir. When adding to this test plan, please
+do not fail to consider how your additions may be automated in the future, as
+this is the end goal of our release testing.
+
+Note that all of these tests must be run _after_ a release branch is created
+in the Mir repository, such that the repository is updated beforehand.
+
+## Setup
+We will run the test plan on the `miral-app` test compositor.
+
+1. Install mir from the release candidate PPA
+    ```sh
+    sudo add-apt-repository --update ppa:mir-team/rc
+    sudo apt install mir-demos mir-platform-graphics-virtual mir-graphics-drivers-desktop mir-test-tools
+    ```
+2. Confirm that you can run `miral-app`
+
+## Tests
+Each test must be performed across a combination of different display
+platforms and different Ubuntu releases. The following matrix provides
+the environments in which we need to test:
+
+|                                | 24.04    | 24.10      |
+|--------------------------------|----------|------------|
+| gbm-kms                        |          |            |
+| eglstream-kms                  |          |            |
+| eglstream-kms + gbm-kms hybrid |          |            |
+| x11                            |          |            |
+| wayland                        |          |            |
+| virtual                        |          |            |
+
+### Platform Selection
+The first set of tests confirms that platforms are selected appropriately.
```

Comment:
The "tests" in this section are not really tests.

Yes, there's a verification step (checking the log), but they are more like the "setup" needed for each row in the above table (for which "the remaining test sections" should be applied).